app/scripts/modules/core/src/managed/ManagedResourceObject.tsx

```diff
@@ -57,20 +57,37 @@ const getResourceRoutingInfo = (
 
 export const ManagedResourceObject = ({ resource, artifact, depth }: IManagedResourceObjectProps) => {
   const { version: currentVersion, buildNumber: currentBuild } = parseName(artifact?.versions.current || '') || {};
+  const { version: deployingVersion, buildNumber: deployingBuild } =
+    parseName(artifact?.versions.deploying || '') || {};
   const { kind } = resource;
   const resourceName = getResourceName(resource);
   const routingInfo = getResourceRoutingInfo(resource) ?? { state: '', params: {} };
   const route = useSref(routingInfo.state, routingInfo.params);
 
+  const currentPill = artifact?.versions.current ? (
+    <Pill text={currentBuild ? `#${currentBuild}` : currentVersion || artifact.versions.current || 'unknown'} />
+  ) : null;
+  const deployingPill = artifact?.versions.deploying ? (
+    <>
+      <Icon appearance="dark" name="caretRight" size="medium" />
```

Comment:
I like how this came out — glad you went with it over a CSS impl.

Given that this is a visually secondary element to other things like the resource icon and version pills, should we go for the `neutral` appearance instead of `dark`?